conbench/db.py

```diff
@@ -39,6 +40,7 @@ def configure_engine(url):
         # https://github.com/conbench/conbench/pull/690
         connect_args={"options": "-c timezone=utc -c statement_timeout=60s"},
     )
+    log.info("bind engine to session")
```

Comment:
When developing this target I needed to see where things hang and why; this helped.